wasm_for_tests/wasm_source/src/lib.rs

```diff
@@ -158,6 +158,21 @@ pub mod main {
     }
 }
 
+#[cfg(feature = "tx_invalid_data")]
+pub mod main {
+    use namada_tx_prelude::*;
+
+    #[transaction(gas = 1000)]
+    fn apply_tx(ctx: &mut Ctx, tx_data: Tx) -> TxResult {
+        let signed = tx_data;
+        let _data = signed.data().ok_or_err_msg("Missing data").map_err(|err| {
+            ctx.set_commitment_sentinel();
```

Comment:
Actually here we might just set the commitment sentinel and return an error all the times, which would avoid the need to create an invalid transaction in the first place

Reply:
Eh, do I have to ???